README.md

```diff
@@ -1,7 +1,26 @@
 
-# penalties
+# Penalties
 
-This is a placeholder README.md for a new repository
+The backend for the penalties services. It handles the following:
+
+- Retrieving penalties and compliance data
+- The retrieval and submission of appeals
+- Providing other teams with penalties summary data 
+
+## Running
+
+The service manager configuration name for this service is `PENALTIES`. 
+But it can be started with`sm --start PENALTIES_ALL` along with dependent services such as `penalties-stub` and more.
+
```

Comment:
Can we mention that we need to run  sm --stop PENALTIES  in order to run it locally